fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -1599,6 +1599,26 @@ private void runAlterJobV2() {
         });
     }
 
+    public List<List<Comparable>> getAllAlterJobInfos() {
+        List<List<Comparable>> schemaChangeJobInfos = new LinkedList<>();
```

Comment:
Same as AlterHandler, no need to check priv

Reply:
fixed